dvc/ignore.py

```diff
@@ -183,12 +193,20 @@ def __init__(self, tree, root_dir):
 
         self.tree = tree
         self.root_dir = root_dir
-        self.ignores_trie_tree = StringTrie(separator=os.sep)
+        self.ignores_trie_tree = PathStringTrie()
         self.ignores_trie_tree[root_dir] = DvcIgnorePatterns(
             default_ignore_patterns, root_dir
         )
+        self._ignored_subrepos = PrefixSet(factory=PathStringTrie)
         self._update(self.root_dir)
 
+    @property
+    def ignored_subrepos(self):
+        # as we go collecting subrepos dynamically, `_ignore_subrepos` will be
+        # populated which is useful for finding out if any subrepos were
+        # encountered along the way
+        return self._ignored_subrepos
```

Comment:
Not sure I follow. Is it a draft for the future functionality? Right not it just acts as a proxy and seems meaningless.